app/views/govuk_publishing_components/components/docs/emergency_banner.yml

```diff
@@ -45,3 +45,13 @@ examples:
       short_description: "1491 to 1547"
       link: "https://www.gov.uk/"
       homepage: true
+  with_ga4_tracking:
+    description: |
+      Toggles emergency_banner: true on the GA4 pageview object, and enables the GA4 link tracker on the banner.
```

Comment:
Nitpick: I'd maybe reword this as `Enables GA4 tracking on the banner. This includes link tracking on the component itself, and allows pageviews to record the presence of the banner on page load.`